backends/gdx-backend-lwjgl3/src/com/badlogic/gdx/backends/lwjgl3/Lwjgl3Application.java

```diff
@@ -76,6 +77,7 @@ public class Lwjgl3Application implements Lwjgl3ApplicationBase {
 	private static GLVersion glVersion;
 	private static Callback glDebugCallback;
 	private final Sync sync;
+	private boolean recreateWindow = false;
```

Comment:
I think this could use a more descriptive name like `currentlyRecreatingWindow`.